power_grid_model_c/power_grid_model/include/power_grid_model/main_model_impl.hpp

```diff
@@ -388,9 +383,12 @@ class MainModelImpl<ExtraRetrievableTypes<ExtraRetrievableType...>, ComponentLis
     }
 
     // Entry point for main_model.hpp
-    template <class... ComponentTypes>
     main_core::utils::SequenceIdx<ComponentType...> get_all_sequence_idx_map(ConstDataset const& update_data) {
-        return main_core::update::get_all_sequence_idx_map<ComponentType...>(state_, update_data);
+        auto const components_to_update = get_components_to_update(update_data);
+        auto const update_independence =
+            main_core::update::independence::check_update_independence<ComponentType...>(state_, update_data);
+        return main_core::update::get_all_sequence_idx_map<ComponentType...>(
+            state_, update_data, 0, components_to_update, update_independence, false);
     }
```

Comment:
Now this function is only called in the restore function in main_model.hpp, hmm

Reply:
that's great news because that functionality is deprecated and removed in #848 😬

Reply:
(or, at least, public exposition of it)

Reply:
mmmk